.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
 - source activate deepchem
 - pip install yapf==0.16.0
 - python setup.py install
+- KERAS_BACKEND=tensorflow
```

Comment:
I think default Keras backend is now tf, so we don't need to explicitly set this IIRC.